### ISSUE TITLE
Error in command  make sysgen 2>&1 | tee build.log

### ISSUE THREAD

**Issue by joselcarrillom** [iraf-community/iraf#81]

**Comment by joselcarrillom** [comment]
Greetings:
I downloaded the package and followed the installing instructions. When I ran the command "make sysgen 2>&1 | tee build.log"  after an extense log I got the following error: 
"make: *** [Makefile:19: sysgen] Error 1"
and when I tried to test the build by running  ./test/run_tests   I got:

IRAF shell ecl.e not found

I would really appreciate a bit of help here, I am following the installing instructions exactly as they are and can't complete the installation after all. For the record my OS is an ArchLinux based (manjaro) Thank you very much in advance for the help.

**Comment by olebole** [comment]
You should search the `build.log` for further error messages. I would guess that the ncurses header files are not found. If you can't find it, please attach the `build.log` to the issue.

**Comment by olebole** [comment]
Does the problem still exist? If not, I would close this issue.

**Comment by AlderaminCph** [comment]
Hello.
I've exactly the same problem.
I've tried to compile IRAF package from github and I've got multiple definition error.
```/usr/bin/ld: xppcode.o:/home/nastya/iraf/unix/boot/spp/xpp/xppcode.c:109: multiple definition of `errflag'; xppmain.o:/home/nastya/iraf/unix/boot/spp/xpp/xppmain.c:21: first defined here
collect2: error: ld returned 1 exit status
make: *** [Makefile:19: sysgen] Error 1
```
My operating system is Manjaro Linux. I use gcc (GCC) 10.1.0.

I've also tried to compile this package on other LINUX distros (on Ubuntu, Debian) and everything was fine.

The problem appears only on Arch. I also tried to use different versions of GCC (5.5 and 7.5.0) and it doesn't solve the problem.

I also attach my build.log file.
I appreciate any help and advice.
[build.log](https://github.com/iraf-community/iraf/files/4741071/build.log)
